src/workloads/query/TelemetryQueryShapes.yml

```diff
@@ -0,0 +1,129 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of telemetry with different query shapes.
+
+KeyWords:
+- telemetry
+- query
+- shape
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
```

Comment:
Do you know why this is here? Or just copied from another test?

Reply:
I was having this error: `Error in parallel execution. First exception's what(): Failed to acquire an entry from the client pool.` And saw this was the fix someone else used for it on slack

Reply:
Weird. Looks like many tests do this without any comment... 🤷🏼

Reply:
Yeah, I saw there is a default limit of 100 for the `maxPoolSize` so I guess for certain workloads that max is reached and needs to be increased?